runelite-client/src/main/java/net/runelite/client/plugins/wiki/WikiSearchChatboxTextInput.java

```diff
@@ -297,7 +299,14 @@ public void keyPressed(KeyEvent ev)
 
 	private void search(String search)
 	{
-		LinkBrowser.browse(WikiPlugin.WIKI_BASE + "?search=" + URLEncoder.encode(search) + "&" + WikiPlugin.UTM_PARAMS);
+		try
+		{
+			LinkBrowser.browse(WikiPlugin.WIKI_BASE + "?search=" + URLEncoder.encode(search, Charset.defaultCharset().name()) + "&" + WikiPlugin.UTM_PARAMS);
+		}
+		catch (UnsupportedEncodingException e)
+		{
+			log.warn("error searching wiki", e);
+		}
 		chatboxPanelManager.close();
```

Comment:
This should be in a `finally` block.
```suggestion
		finally
		{
			chatboxPanelManager.close();
		}
```

Reply:
no it should not

Reply:
Agreed, there is no reason to have a finally block here.